RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,179 @@
+  subsets of the DAN, called VN committees. These committees reach consensus on DA state amongst themselves.
+* VNs earn fees for their efforts.
+* Digital asset contracts are not Turing complete, but are instantiated by [Asset Issuer]s (AIs) using Digital Asset templates that are defined
+  in the DAN protocol code.
+
+### Digital Assets
+
+* Digital asset contracts are *not* Turing complete, but are selected from a set of [DigitalAssetTemplate]s that govern
+  the behaviour of each contract type. e.g. there could be a Single-Use Token template for simple ticketing systems; a
+  Coupon template for loyalty programmes and so on.
+* The template system is intended to be highly flexible and additional templates can be added to the protocol periodically.
+* Asset issuers can register TLDIs (top-level digital issuer) names on the base chain to help disambiguate similar
+  contracts and improve the signal-to-noise ratio from scam- or copy-cat contracts.
+
+An [Asset Issuer] (AI) will issue a Digital Assets by constructing a contract from on of the supported set of [DigitalAssetTemplate]s. The AI will choose
+ how large the committee of Validator Nodes will be for this DA and have the option to nominate [Trusted Node]s to be part of the VN committee for the DA.
+Any remaining spots on the committee will be filled by permissionless VNs that are selected randomly by the DAN.
+
+### Validator Nodes
+
+[Validator Node]s form the basis of the second layer DAN. All actions on this network take place by interacting with VN's. Some examples of actions
+that VNs will facilitate are:
+* Issuing a [Digital Asset],
+* querying the state of [Digital Asset] and its constituent [tokens],
+* issuing and instruction to change the state of a [Digital Asset] or [tokens].
+
+VNs will also perform archival function for the assets they manage. The lifetime of these archives and the fee structure for this function is
+still being discussed.
+
+#### Registration
+VNs register themselves on the [Base Layer] using a special [transaction] type. The registration [transaction] type
+requires spending of a certain minimum amount of [Tari coin], the ([RegistrationCollateral]), that has a time-lock on the
+output for a minimum amount of time ([RegistrationPeriod]) as well as some metadata, such as the VNs public key and a generated Node ID. The
+details of this transaction are not confidential and are publicly auditable. The Node ID is generated during registration to prevent mining
+of VN public keys that can be used to manipulate routing on the DAN.
+
+VNs may spend this Tari back to themselves which will end their registration.
+
+Requiring nodes to register themselves serves two purposes:
+* Makes VN Sybil attacks expensive,
+* Provides an authoritative "central-but-not-centralised" registry of validator nodes from the base layer.
+
+#### Validation of instructions
```

Comment:
What do you think about calling this `execution` rather than validation, given what VNs actually do?

Reply:
Agreed.